library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -77,11 +77,15 @@ public void onMapReady(GoogleMap googleMap) {
   }
 
   @Override public void addMarker(AirMapMarker airMarker) {
-    airMarker.addToGoogleMap(googleMap);
+    Marker marker = googleMap.addMarker(airMarker.getMarkerOptions());
+    airMarker.setMarker(marker);
```

Comment:
im not a big fan of this because it makes `AirMapMarker` mutable and makes it know about `Marker` but we can improve this in a future PR